bids-hook.go

```diff
@@ -457,20 +456,17 @@ func readConfig() {
 	}
 	giteaToken = []byte(val)
 
-	val, ok = os.LookupEnv("GITEA_PUBLIC_PATH")
+	val, ok = os.LookupEnv("GITEA_CUSTOM")
 	if !ok {
-		log.Fatal("missing environment variable GITEA_PUBLIC_PATH")
+		log.Fatal("missing environment variable GITEA_CUSTOM")
 	}
-	giteaPublicPath, err = filepath.Abs(val)
+	giteaCustom, err = filepath.Abs(val)
 	if err != nil {
-		log.Fatalf("invalid GITEA_PUBLIC_PATH: %v", err)
+		log.Fatalf("invalid GITEA_CUSTOM: %v", err)
 	}
-	info, err = os.Stat(giteaPublicPath)
+	err = os.MkdirAll(filepath.Join(giteaCustom, "public"), 0750)
 	if err != nil {
-		log.Fatalf("error opening GITEA_PUBLIC_PATH: %v", err)
-	}
-	if !info.IsDir() {
-		log.Fatal("GITEA_PUBLIC_PATH is not a directory")
+		log.Fatalf("error creating output folder: %v", err)
```

Comment:
I'm doing this here so I don't have to do it in ansible. It's a lot easier if bids-hook can be relatively self-contained.